style.css

```diff
@@ -0,0 +1,86 @@
+h1{	
+		font-size: 80px;
+		text-align: center; 
+		color: Yellow;
+		margin-bottom: -10px;
+		text-decoration: underline;
+
+	}
+
+h3{
+		text-align: center; 
+		color: blue;
+	}
+
+h3.flea {
+		text-align:left;
+}	
+
+p {
+		margin-left: 30px;
+		margin-right: 20px;
+		line-height: 25px;
+		margin-bottom: 50px;
+	}
+
+body {
+
+		font-family: sans-serif ;
+
+		background-color:#D2D2D2;
+		color:black;
+		margin-left: 30px;
+		margin-bottom: 10px;
+		margin-right: 30px;
+	}
+
+img.action  {	
+			border: solid 5px;
+			float: right;
+			width: 450px;
+			height: 550px;
+			margin-left: 75px;
+			margin-right: 50px
+	}
+
+img.main {	
+		border: solid 5px;
+		float: center;
+		margin-left: 230px;
+		margin-top: 20px;
+	}
+
+nav {
+		float: center;
```

Comment:
:hammer: Center isn't a valid property for float. Did you mean `text-align: center;`?